HVSample/src/main/res/layout/activity_fileupload.xml

```diff
@@ -1,42 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical" >
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:orientation="vertical" >
 
-    <Button
-        android:id="@+id/defaultFileupload"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Default"
-        android:layout_gravity="center_horizontal"
-        android:layout_marginTop="10dp" />
-    
-    <FrameLayout
-        android:layout_width="match_parent"
-        android:layout_height="fill_parent" >
-    <LinearLayout android:id="@+id/camera_preview"
-    android:orientation="horizontal"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <SurfaceView android:id="@+id/camera_surfaceView"
-      android:layout_weight="1"
-      android:layout_width="0dp"
-      android:layout_height="match_parent"/>
-    <Button android:id="@+id/camera_click"
-      android:layout_width="wrap_content"
-      android:layout_height="match_parent"
-      android:text="click"/>
+	<Button
+		android:id="@+id/defaultFileupload"
+		android:layout_width="wrap_content"
+		android:layout_height="wrap_content"
+		android:text="Default"
+		android:layout_gravity="center_horizontal"
+		android:layout_marginTop="10dp" />
+
+	<FrameLayout
+		android:layout_width="match_parent"
+		android:layout_height="fill_parent" >
+	<LinearLayout android:id="@+id/camera_preview"
+	android:orientation="horizontal"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent">
+	<SurfaceView android:id="@+id/camera_surfaceView"
+	  android:layout_weight="1"
+	  android:layout_width="0dp"
+	  android:layout_height="match_parent"/>
+	<Button android:id="@+id/camera_click"
+	  android:layout_width="wrap_content"
+	  android:layout_height="match_parent"
+	  android:text="click"/>
```

Comment:
nit: Do we just want a button that says click? Is there not a image resource of a shutter button or something we can use to make it look a little better or do we not care about style for the sample?